### DISCUSSION TITLE
How to deal with bibliography files?

### DISCUSSION THREAD

**Discussion by svraka** [ropensci/tarchetypes#73] [Show and tell]
Thinking about my previous question in https://github.com/ropensci/tarchetypes/discussions/70 it occurred to me that there's another area, where a dependency is not discovered: bibliography files. These are typically set in the YAML header (e.g. `bibliography: literature.bib`). These can be dealt with by adding a file target to the bibliography file and then using `tar_load()` on that in the R Markdown document. 

I don't think **targets** needs to provide any functionality to automate this (and some R Markdown templates might use other metadata fields for similar file dependencies), I'm only bringing this up in case someone finds it useful.

**Comment by wlandau** [comment]
Nice! I like this solution.
